dmutils/__init__.py

```diff
@@ -2,7 +2,7 @@
 import flask_featureflags
 from flask_featureflags.contrib.inline import InlineFeatureFlag
 
-__version__ = '2.0.4'
+__version__ = '2.0.5'
```

Comment:
Needs to be 2.1.1 now I think. (Sorry, broke your mergeability by merging Paul's PR)